modules/fts/pages/fts-index-type.adoc

```diff
@@ -0,0 +1,33 @@
+== Index Type
+The *Index Type* interface provides a pull-down menu, from which the appropriate index type can be selected:
+
+[#index_type_interface_image]
+image::fts-index-type-interface.png[,%100]
+
+Two options are available: 
+
+** Version 6.0 (Scorch) reduces the size of the index-footprint on disk, and provides enhanced performance for indexing and mutation-handling
+
+** Version 5.0 (Moss) is the standard form of index, to be used in test, development, and production
```

Comment:
Moss is now deprecated, and I expected to see this mentioned here.

Reply:
Update: add (Deprecated)